packaging/suse/trento-agent.spec

```diff
@@ -33,7 +33,9 @@ BuildRequires:  golang(API) = 1.16
 Requires:       golang-github-prometheus-node_exporter
 Provides:       %{name} = %{version}-%{release}
 Provides:       trento = %{version}-%{release}
+Provides:       trento-premium = %{version}-%{release}
 Obsoletes:      trento < %{version}-%{release}
```

Comment:
Now that I think about this, I suspect we should have probably not used the macro here to fix the (deprecated) provided version to an specific version.